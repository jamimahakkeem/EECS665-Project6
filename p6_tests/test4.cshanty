void main() {
	int a;
	a = 5;
	bool b;
	b = a < 1;
	b = a <= 1;
	b = a > 1;
	b = a >= 1;
	b = a == 1;
	b = a != 1;
<<<<<<< HEAD
}
=======
}
>>>>>>> jamima1
