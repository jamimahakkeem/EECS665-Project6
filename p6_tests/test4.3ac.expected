[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN main LOCALS]
a (local var of 8 bytes)
b (local var of 8 bytes)
tmp0 (tmp var of 8 bytes)
tmp1 (tmp var of 8 bytes)
tmp2 (tmp var of 8 bytes)
tmp3 (tmp var of 8 bytes)
tmp4 (tmp var of 8 bytes)
tmp5 (tmp var of 8 bytes)
[END main LOCALS]
main:       enter main
            [a] := 5
            [tmp0] := [a] LT64 1
            [b] := [tmp0]
            [tmp1] := [a] LTE64 1
            [b] := [tmp1]
            [tmp2] := [a] GT64 1
            [b] := [tmp2]
            [tmp3] := [a] GTE64 1
            [b] := [tmp3]
            [tmp4] := [a] EQ64 1
            [b] := [tmp4]
            [tmp5] := [a] NEQ64 1
            [b] := [tmp5]
lbl_0:      leave main
